CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unversioned
+### Changed
+- The Backbone router was replaced by the react-router in order to remove the
+  jQuery and Backbone dependency completely.
```

Comment:
Should we write a short sentence about what effect this has for the users? I.e. the new URI schemes?